Subject: Add documentation field to the systemd unit
Author: Apollon Oikonomopoulos <apoikos@gmail.com>

Forwarded: no
Last-Update: 2014-01-03
--- a/contrib/systemd/haproxy.service.in
+++ b/contrib/systemd/haproxy.service.in
@@ -1,5 +1,7 @@
 [Unit]
 Description=HAProxy Load Balancer
+Documentation=man:haproxy(1)
+Documentation=file:/usr/share/doc/haproxy/configuration.txt.gz
 After=network.target syslog.service
 Wants=syslog.service
 
